frontend/src/App.tsx

```diff
@@ -22,15 +25,21 @@ function AppWithRedux(props: React.PropsWithChildren<{}>) {
     initializePlugins();
   }, [themeName]);
 
-  return <ThemeProviderNexti18n theme={themes[themeName]}>{props.children}</ThemeProviderNexti18n>;
+  return (
+    <I18nextProvider i18n={i18n}>
+      <ThemeProviderNexti18n theme={themes[themeName]}>{props.children}</ThemeProviderNexti18n>
+    </I18nextProvider>
+  );
 }
 
 function App() {
   return (
     <Provider store={store}>
-      <AppWithRedux>
-        <AppContainer />
-      </AppWithRedux>
+      <React.Suspense fallback={<Loader title="Loading…" />}>
```

Comment:
I think React.Suspense is still an experimental feature (https://reactjs.org/docs/concurrent-mode-suspense.html). We had trouble with it long ago too (I seem to recall).

Reply:
I have figured it out in the end without having to use the suspend.